homeassistant/components/imap/coordinator.py

```diff
@@ -209,6 +209,30 @@ def text(self) -> str:
         return str(self.email_message.get_payload())
 
 
+@callback
+def get_parts(message: Message, prefix: str | None = None) -> dict[str, Any]:
+    """Return information about the parts of a multipart message."""
+    index = 0
+    parts: dict[str, Any] = {}
+    if not message.is_multipart():
+        return {}
+    for part in message.get_payload():
+        if TYPE_CHECKING:
+            assert isinstance(part, Message)
+        key = f"{prefix},{index}" if prefix else f"{index}"
+        index += 1
```

Comment:
Use `enumerate` instead of this manual loop counter

Reply:
Fixed